src/AITelemetry.js

```diff
@@ -0,0 +1,28 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ReactPlugin } from '@microsoft/applicationinsights-react-js';
+
```

Comment:
Just nitpicky but I'd call this `TelemetryService` or something so the calls below don't feel ambiguous. 

e.g. `telemetryService.AI.track...`